integration/internal/exec/exec.go

```diff
@@ -0,0 +1,85 @@
+package exec
+
+import (
+	"bytes"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+	"github.com/docker/docker/pkg/stdcopy"
+	"golang.org/x/net/context"
+)
+
+type ExecResult struct {
+	ExitCode  int
+	outBuffer *bytes.Buffer
+	errBuffer *bytes.Buffer
+}
+
+func (res *ExecResult) Stdout() string {
+	return res.outBuffer.String()
+}
+
+func (res *ExecResult) Stderr() string {
+	return res.errBuffer.String()
+}
+
+func (res *ExecResult) Combined() string {
+	return res.outBuffer.String() + res.errBuffer.String()
+}
+
+// ExecSimple executes a command inside a container, returning the result
+// containing stdout, stderr, and exit code.
+func ExecSimple(ctx context.Context, cli client.APIClient, id string, cmd []string) (ExecResult, error) {
+	// Check that container is up
+	if _, err := cli.ContainerInspect(ctx, id); err != nil {
```

Comment:
This doesn't check if the container is running, just that it exists. Is this necessary or will execCreate handle this check for us?

Reply:
I wrote it in November so frankly I don't remember :) but it looks like this is indeed a check that container exists, and I think it can be dropped.